apis/nucleus/src/components/listbox/components/ListBoxRowColumn/ListBoxRowColumn.jsx

```diff
@@ -102,8 +102,18 @@ function RowColumn({ index, rowIndex, columnIndex, style, data }) {
   }, [rowRef, focusListItems.first, focusListItems.last]);
 
   const handleKeyDownCallback = useCallback(
-    getRowsKeyboardNavigation({ ...actions, focusListItems, keyboard, isModal }),
-    [actions, keyboard?.innerTabStops]
+    getRowsKeyboardNavigation({
+      ...actions,
+      focusListItems,
+      keyboard,
+      isModal,
+      rowCount,
+      columnCount,
+      rowIndex,
+      columnIndex,
+      layoutOrder,
+    }),
+    [actions, keyboard?.innerTabStops, rowCount, columnCount, rowIndex, columnIndex, layoutOrder]
```

Comment:
Don't know if needed… but might have to add `focusListItems` too:
```suggestion
    [actions, focusListItems, keyboard?.innerTabStops, rowCount, columnCount, rowIndex, columnIndex, layoutOrder]
```